src/Kiosk.Abstractions/Models/LessonPlan/LessonPlanResponse.cs

```diff
@@ -0,0 +1,35 @@
+using System.Text.Json.Serialization;
+using Kiosk.Abstractions.Enums;
+
+namespace Kiosk.Abstractions.Models.LessonPlan;
+
+public class LessonPlanResponse
```

Comment:
you probably could change this dto name to something like `FindLessonPlanResponse` or `GetLessonPlanResponse` since you use it for get requests